<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GradientPageClass</class>
 <widget class="QWidget" name="GradientPageClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>205</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,1,1,1">
     <item>
      <widget class="BtnCtrl" name="RGBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>R.Gradient1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="BtnCtrl" name="RGBtn2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>R.Gradient2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="BtnCtrl" name="RGBtn3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>R.Gradient3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="BtnCtrl" name="RGBtn4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>R.Gradient4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="BtnCtrl" name="RGBtn5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>PushButton5</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1,1,1,1">
     <item>
      <widget class="BtnCtrl" name="RGBtn6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>R.Gradient6</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="BtnCtrl" name="RGBtn7">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>R.Gradient7</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="BtnCtrl" name="RGBtn8">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>R.Gradient8</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="BtnCtrl" name="RGBtn9">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>R.Gradient9</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="BtnCtrl" name="RGBtn10">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>R.Gradient10</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4" stretch="2,3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,1">
       <item>
        <widget class="BtnCtrl" name="FGBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>F.Gradient</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="BtnCtrl" name="CGBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>C.Gradient</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BtnCtrl</class>
   <extends>QPushButton</extends>
   <header>btnctrl.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
